osidb/models/flaw/label.py

```diff
@@ -175,6 +176,15 @@ class Meta:
                 fields=["flaw", "label"], name="unique label per flaw"
             ),
         ]
+        ordering = [
+            models.Case(
+                models.When(type=FlawLabel.FlawLabelType.ALIAS, then=0),
+                models.When(type=FlawLabel.FlawLabelType.CONTEXT_BASED, then=1),
+                models.When(type=FlawLabel.FlawLabelType.PRODUCT_FAMILY, then=2),
+                default=3,
+            ),
+            "label",
+        ]
```

Comment:
IMO there's no reason to order the labels in the backend, especially when it's so arbitrary.

It's highly unlikely/impossible (this was not captured in the ticket but from the refinement call) for there to be more than one alias label per flaw, the frontend can just search for the label of type alias and always display it first if they so desire.

But I'm willing to go with this approach if @MrMarble or @C-Valen have a different opinion